claims_hosp/HospClaims/automate/regen_old_issue.py

```diff
@@ -0,0 +1,47 @@
+from datetime import datetime, timedelta
```

Comment:
For some context, this file is called whenever I want to generate an older issue by "faking" the current time of the system.